api/src/wfl/api/handlers.clj

```diff
@@ -102,6 +102,8 @@
   (log/info (select-keys request [:request-method :uri :parameters]))
   (let [uuid   (get-in request [:path-params :uuid])
         status (get-in request [:body-params :status])]
+    ;; TODO: if status='Succeeded', no op / error?
```

Comment:
The ticket had this recommendation:

"Consider no-oping for retries where the status is “Succeeded” to avoid workflow-launcher blowing up."

I wanted to see if we wanted to throw an error in this case?  Same question if no workflows are found for the status.

Reply:
Ticketed separately: https://broadinstitute.atlassian.net/browse/GH-1424